README.md

```diff
@@ -222,58 +220,191 @@ example of an 'edgy' configuration.
 
 Plugin exposes following commands:
 
-#### `OGPT`
-`OGPT` command which opens interactive window using the  model in the `config.api_params`
-(also known as `OGPT`)
-
-#### `OGPTActAs`
+### `OGPT`
+`OGPT` command opens an interactive window to communicate with LLM backends. The interactive window consists of four panes:
+* OGPT Chat: Input text area to write prompts.
+* OGPT: Output text area printing responses from LLM.
+* OGPT Parameters: (Toggled by \<Ctrl-o\>,) Parameter panel to display and modify LLM paramters.
+* OGPT Sessions: (Toggled by \<Ctrl-o\>,) Sessions panel to display chat sessions.
+
+#### Useful shortcuts
+<table>
```

Comment:
If anyone knows how to format a table with merged rows in markdown, please do leave a comment. I tried the markdown table with row span in line 324, but it didn't work as I wished.